Dockerfile.prod

```diff
@@ -16,6 +16,7 @@ FROM node:14.17-alpine
 USER node
 COPY --chown=node:node --from=build /app /app
 WORKDIR /app
-ENV NODE_ENV development
+ENV NODE_ENV production
 
-CMD ["npm", "run", "start:production"]
+RUN chmod +x ./bin/start-server-micros.sh
```

Comment:
the file should already be executable, you don't need to add this in the dockerfile.  If it's not, make it so and commit/push the file.